### ISSUE TITLE
Thick Subtitles Border/Outline

### ISSUE THREAD

**Issue by Tama47** [mpv-player/mpv#11071]
### Important Information

Provide following Information:
- mpv 0.35.0
- macOS Ventura Version 13.1
- Homebrew Formulae

### Reproduction steps

Something happened to the recent version of mpv. Normally, srt and vtt subtitles are shown without thick border, but even with --no-config the subtitles still shows thick outline. Now, when I use sub-ass-override to change the subtitles to my preferred styles, it shows even thicker outline. This used to not be the case.

--no-config
![mpv-shot0003](https://user-images.githubusercontent.com/96937943/210157874-8a2ab6db-f3f2-44e8-9a8c-de0f5c9c95e9.jpg)

### Expected behavior
![mpv-shot0001](https://user-images.githubusercontent.com/96937943/210157737-16ee69ce-222b-4070-a14d-90585209dc30.png)

### Actual behavior
![mpv-shot0002](https://user-images.githubusercontent.com/96937943/210157738-2c386053-8b38-47e1-a325-4f7db6bbb58f.png)

### Log file

[output.txt](https://github.com/mpv-player/mpv/files/10328135/output.txt)

### Sample files

```bash
# Sub #
blend-subtitles=video
slang=en,en-US,eng,english-subs,jpn,th,tha
sub-ass-force-style=ScaledBorderAndShadow=yes
sub-auto=fuzzy
sub-border-color="#000000"
sub-border-size=4
sub-color="#FFFFFFFF"
sub-font-size=48
sub-font=Trebuchet MS
sub-margin-y=36
sub-shadow-color="#000000"
sub-shadow-offset=2
```

**Comment by Inclushe** [comment]
Getting this issue building from homebrew and source (Waf). Using a precompiled Intel version or [IINA](https://github.com/iina/iina) works for now.

Precompiled Intel mpv:
![mpv-shot0001](https://user-images.githubusercontent.com/7546357/225488473-881541ff-0bfd-4d2c-8a80-261fe4e5b9c5.jpg)

Built from source mpv:
![mpv-shot1253](https://user-images.githubusercontent.com/7546357/225488555-fec4d52e-fdfd-4334-9bb1-2c42232d6a1b.jpg)

**Comment by Tama47** [comment]
Thanks for the insights! So is the issue with the source? That's unfortunate because I prefer using the Homebrew native Apple Silicon build.

**Comment by llyyr** [comment]
This is probably because of changes in libass (https://github.com/libass/libass/pull/645) to make it have the same behavior as xysubfilter

**Comment by Inclushe** [comment]
IINA updated to the newest mpv build and now has this same issue. Cycling through aspect ratios with the keyboard shortcut `A` causes subtitle borders to stretch according to the aspect ratio.

This is being discussed in #11825 and a pull request with the fix (#11831) is currently open.
I built the files in [the `fix-srt` branch](https://github.com/llyyr/mpv/tree/fix-srt) and the issue has been fixed for my local install.
